lib/flipper/utilities/bulk_feature_checker.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Flipper
+  module Utilities
+    module BulkFeatureChecker
+      # Accepts an array of feature keys (symbol or string format) and returns a result hash of which keys
+      # are enabled, disabled, or missing; feature keys are returned in symbol format
+      def self.status(features)
+        result = {
+          enabled: [],
+          disabled: [],
+          missing: []
+        }
+
+        Array(features).map(&:to_sym).each do |feature|
+          unless feature_exists?(feature)
+            result[:missing] << feature
+            next
+          end
+
+          if Flipper.enabled?(feature)
+            result[:enabled] << feature
+          else
+            result[:disabled] << feature
+          end
+        end
+
+        result
+      end
+
+      def self.feature_exists?(feature)
+        Flipper.features.map(&:name).include?(feature)
```

Comment:
Doing this call & map for every iteration within the `features.each` loop is nesting loops unnecessarily - even if `Flipper.features` weren't also potentially reading its config file / making a call to the database / or pinging redis every time it's called.

`include?` seems clear enough to me to use in the `status` method. Let's just call `Flipper.features.map(&:name)` once and store it into a variable, then
```ruby
result[:missing] << feature if existing_features.include?(feature)
```